app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -47,8 +50,8 @@ def index
     roles = UserRole.filter_roles(roles, current_user, params)
     roles = UserRole.sort_roles(roles, params[:sort])
 
-    # Limiting to first 100 elements of roles array to avoid serializing of large array.
-    render json: roles.first(100)
+    # Paginating the list by sending only first 100 elements unless mentioned in the API.
+    paginate json: roles
```

Comment:
This change seems like it was included in your previous pagination PR.
Is there maybe a chance you didn't include the latest changes from `main` in your currentPR branch?

Reply:
Oops I don't know how this came, removed now.